core/block/template/service.go

```diff
@@ -161,7 +161,8 @@ func (s *service) ObjectApplyTemplate(contextId, templateId string) error {
 		layout, found := orig.Layout()
 		if found {
 			if commonOperations, ok := b.(basic.CommonOperations); ok {
-				if err = commonOperations.SetLayoutInStateAndIgnoreRestriction(ts, layout); err != nil {
+				ts.RemoveDetail(bundle.RelationKeyLayout.String())
```

Comment:
We need to remove Layout detail here, because **ts** has the same details with **orig**, so SetLayoutInState actually did no changes on blocks content